client/src/App.js

```diff
@@ -24,7 +24,7 @@ function App() {
                         <Route exact path="/sign-up" component={Signup} />
```

Comment:
Line 23: consider what having `exact` will do when you're trying to access the Profile page

Reply:
5 characters that made a huge difference!